harvard-lts/CURIOSity
# K8s final

**TK8s Changes.**
* * *

# What does this Pull Request do?
This PR allows for a few changes:
- makes the configurable stdout logging be respected by dev environment and worker (required for k8s)
- removes internal https (required for k8s)
- removes unused n8 ci stuff
- updates rails for security patches
- changes the healthcheck to the node healthcheck for swarm and (I think) removes curl

# How should this be tested?

This is on k8s dev as proof it works for k8s. You should be able to build and run locally without issues (note that you now go to a non https local address). There is a PR that will need to be run on Curiosity QA (and once merged will be implemented in the next prod release): https://github.huit.harvard.edu/LTS/CURIOSity-IF/pull/37

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
